skbio/util/_metadata_repr.py

```diff
@@ -0,0 +1,171 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+from future.utils import with_metaclass
+
+import six
+import itertools
+import numbers
+import textwrap
+
+from abc import ABCMeta, abstractmethod
+from skbio.sequence._base import ElasticLines
```

Comment:
Can you move `ElasticLines` into `skbio._base` (as well as the unit tests)? It's being used now within `sequence` and `alignment`.